docs/quickstart.md

```diff
@@ -1,150 +1,160 @@
+⠿ Container mysql                      Running                                                                                                                                                         0.0s
+⠿ Container datahub-gms                Running                                                                                                                                                         0.0s
+⠿ Container mysql-setup                Started                                                                                                                                                         0.7s
+⠿ Container datahub-datahub-actions-1  Running                                                                                                                                                         0.0s
+⠿ Container datahub-frontend-react     Running                                                                                                                                                         0.0s
+.......
+✔ DataHub is now running
+Ingest some demo data using `datahub docker ingest-sample-data`,
+or head to http://localhost:9002 (username: datahub, password: datahub) to play around with the frontend.
+Need support? Get in touch on Slack: https://slack.datahubproject.io/
+```
 
 :::note
 
-If you've enabled [Metadata Service Authentication](authentication/introducing-metadata-service-authentication.md), you'll need to provide a Personal Access Token
-using the `--token <token>` parameter in the command.
+On Mac computers with Apple Silicon (M1, M2 etc.), you might see an error like `no matching manifest for linux/arm64/v8 in the manifest list entries`.
+This typically means that the datahub cli was not able to detect that you are running it on Apple Silicon.
+To resolve this issue, override the default architecture detection by issuing `datahub docker quickstart --arch m1`
 
 :::
 
-That's it! Now feel free to play around with DataHub!
-
-## Troubleshooting Issues
+### Signing In
 
-Please refer to [Quickstart Debugging Guide](./troubleshooting/quickstart.md).
-
-## Next Steps
+Upon completion of this step, you should be able to navigate to the DataHub UI at [http://localhost:9002](http://localhost:9002) in your browser.
+You can sign in using the default credentials below.
 
-### Ingest Metadata
-
-To start pushing your company's metadata into DataHub, take a look at [UI-based Ingestion Guide](./ui-ingestion.md), or to run ingestion using the cli, look at the [Metadata Ingestion Guide](../metadata-ingestion/README.md).
+```json
+username: datahub
+password: datahub
+```
 
-### Invite Users
+To change the default credentials, please refer to [Change the default user datahub in quickstart](authentication/changing-default-credentials.md#quickstart).
 
-To add users to your deployment to share with your team check out our [Adding Users to DataHub](authentication/guides/add-users.md)
+### Ingesting Sample Data
```

Comment:
small style thing - I kinda prefer headers like `Ingest Sample Data` rather than the progressive `Ingesting`